schedules/backfill_schedule.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+from datetime import datetime, timedelta
+
+from temporalio.client import Client, ScheduleBackfill, ScheduleOverlapPolicy
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_schedule_handle(
+        "workflow-schedule-id",
+    )
+    now = datetime.utcnow()
+    if now.second == 0:
+        now += timedelta(seconds=1)
```

Comment:
Why make this change? We should be wary of blindly copying test code as sample code. We need to make sure we understand every line we are writing as samples.